--- dbus-python-1.2.0/Makefile.am~	2012-10-16 18:38:42.000000000 +0200
+++ dbus-python-1.2.0/Makefile.am	2015-03-11 00:52:36.661397300 +0100
@@ -137,7 +137,6 @@
 	mkdir api
 	cd $(abs_top_srcdir) && \
 	PYTHONPATH=$(APIDOC_PYTHONPATH) DBUS_PYTHON_NO_DEPRECATED=1 \
-	$(PYTHON) -Wignore::DeprecationWarning \
 	$(EPYDOC) -o $(abs_top_builddir)/api --html \
 		--docformat restructuredtext -v \
 		`find dbus -name '*.py' \
